browser-build.config.js

```diff
@@ -0,0 +1,51 @@
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+ module.exports = {
+  entry: "./src/index.js",
+  output: {
+      filename: "geostyler.js",
+      path: __dirname + "/browser",
```

Comment:
Please add the `browser` directory to the [files entry](https://github.com/terrestris/geostyler/blob/master/package.json#L6) in the `package.json`, otherwise it wouldn't be copied to npm.

Reply:
And also add the path to the `.gitignore`.